components/camera/videosource/server_test.go

```diff
@@ -90,16 +90,15 @@ func TestServerSource(t *testing.T) {
 	test.That(t, err, test.ShouldBeNil)
 	test.That(t, img, test.ShouldResemble, rimage.ConvertImage(imgDecode))
 
-	lazyCtx := gostream.WithMIMETypeHint(context.Background(), utils.WithLazyMIMEType(utils.MimeTypePNG))
```

Comment:
Can changes in this file be reverted, and instead just remove the extra arguments to NewLazyEncodedImage()?